config.yaml

```diff
@@ -0,0 +1,420 @@
+---
+imports:
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/iam_service_account_v1/iam_service_account_v1.py
+    name: iam_service_account_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/gce_instancetemplate_v1/gce_instancetemplate_v1.py
+    name: gce_instancetemplate_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/cloudsql_instance_v1/cloudsql_instance_v1.py
+    name: cloudsql_instance_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/address_v1/address_v1.py
```

Comment:
Use `compute.v1.address` or `compute.v1.globalAddress` instead of this custom template.

Reply:
Updated